src/interactions/hoverInteraction.ts

```diff
@@ -34,6 +34,7 @@ export module Interaction {
   export class Hover extends Interaction.AbstractInteraction {
     public _componentToListenTo: Hoverable;
     private _dispatcher: Dispatcher.Mouse;
+    private _touchDispatcher: Dispatcher.Touch;
```

Comment:
We have to do this to restore the initial capabilities that `Hover` had. It acquired touch-responsiveness due to the fact that some Touch events are inherently re-thrown as mousemove events...